src/oatpp/core/data/stream/ChunkedBuffer.hpp

```diff
@@ -36,7 +36,7 @@ namespace oatpp { namespace data{ namespace stream {
 /**
  * Buffer wich can grow by chunks and implements &id:oatpp::data::stream::ConsistentOutputStream; interface.
  */
-class ChunkedBuffer : public oatpp::base::Countable, public ConsistentOutputStream, public std::enable_shared_from_this<ChunkedBuffer> {
+class ChunkedBuffer : public oatpp::base::Countable, public BufferedOutputStream, public std::enable_shared_from_this<ChunkedBuffer> {
```

Comment:
I propose to consider dropping `ChunkedBuffer` class and replace it completely with `BufferOutputStream`

Reply:
Do you mean to drop `ConsistentOutputStream` and refactor its functions to `BufferOutputStream`? It has all those `writeAsString(xxx)` helper functions.

Thats what I was thinking to do in the first place but found that you had a special use-case for `ConsistentOutputStream` and did not want to touch it before you reviewing.

Reply:
No, just to drop `ChunkedBuffer `.
`ConsistentOutputStream` is needed for serializer. Serializers work only with `ConsistentOutputStream` at the moment.